hack/README.md

```diff
@@ -0,0 +1,64 @@
+# Useful resources for development on `lakefs-spec`
+
+This document contains information on the resources in this directory, and how they can be used in development and testing.
+
+## `docker-compose.yml` - local lakeFS quickstart instance
+
+This Docker Compose file bootstraps a local [lakeFS quickstart instance](https://docs.lakefs.io/quickstart/launch.html).
+It does not come with an associated volume, so you can spin it up and down without creating dangling resources.
+
+Requirements:
+* A Docker runtime & CLI with `docker compose`.
+
+To bootstrap, run the following command:
+
+```shell
+docker compose -f hack/docker-compose.yml up
+```
+
+To stop the container again, run
```

Comment:
When you do `docker compose up`, you're attached to the output of the containers, exiting with `Ctrl-C` already stops them, making `docker compose down` unnecessary.